packages/yoroi-extension/app/components/wallet/send/WalletSendFromSteps/WalletSendPreviewStep.js

```diff
@@ -0,0 +1,387 @@
+        await this.props.onSubmit(transactionData);
+      },
+      onError: () => {}
+    });
+  }
+
+  convertedToUnitOfAccount: (TokenEntry, string) => string = (token, toCurrency) => {
+    const tokenInfo = this.props.getTokenInfo(token);
+
+    const shiftedAmount = token.amount
+      .shiftedBy(-tokenInfo.Metadata.numberOfDecimals);
+
+    const coinPrice = this.props.getCurrentPrice(
+      tokenInfo.Identifier,
+      toCurrency
+    );
+
+    if (coinPrice == null) return '-';
+
+    return calculateAndFormatValue(
+      shiftedAmount,
+      coinPrice
+    );
+  }
+
+  renderSingleAmount: TokenEntry => Node = (entry) => {
+    const formatValue = genFormatTokenAmount(this.props.getTokenInfo);
+
+    const { unitOfAccountSetting } = this.props;
+    return unitOfAccountSetting.enabled
+      ? (
+        <>
+          <div className={styles.amount}>
+            {this.convertedToUnitOfAccount(entry, unitOfAccountSetting.currency)}
+            <span className={styles.currencySymbol}>
+              &nbsp;{unitOfAccountSetting.currency}
+            </span>
+          </div>
+          <div className={styles.amountSmall}>{formatValue(entry)}
+            <span className={styles.currencySymbol}>&nbsp;{
+              truncateToken(getTokenName(this.props.getTokenInfo(entry)))
+            }
+            </span>
+          </div>
+        </>
+      ) : (
+        <div className={styles.amount}>{formatValue(entry)}
+          <span className={styles.currencySymbol}>&nbsp;{
+            truncateToken(getTokenName(this.props.getTokenInfo(entry)))
+          }
+          </span>
+        </div>
+      );
+  }
+  renderTotalAmount: TokenEntry => Node = (entry) => {
+    const formatValue = genFormatTokenAmount(this.props.getTokenInfo);
+
+    const { unitOfAccountSetting } = this.props;
+    return unitOfAccountSetting.enabled
+      ? (
+        <>
+          <div className={styles.totalAmount}>
+            {this.convertedToUnitOfAccount(entry, unitOfAccountSetting.currency)}
+            <span className={styles.currencySymbol}>
+              &nbsp;{unitOfAccountSetting.currency}
+            </span>
+          </div>
+          <div className={styles.totalAmountSmall}>{formatValue(entry)}
+            <span className={styles.currencySymbol}>&nbsp;{
+              truncateToken(getTokenName(this.props.getTokenInfo(entry)))
+            }
+            </span>
+          </div>
+        </>
+      ) : (
+        <div className={styles.totalAmount}>{formatValue(entry)}
+          <span className={styles.currencySymbol}>&nbsp;{
+            truncateToken(getTokenName(this.props.getTokenInfo(entry)))
+          }
+          </span>
+        </div>
+      );
+  }
+  renderSingleFee: TokenEntry => Node = (entry) => {
+    const formatValue = genFormatTokenAmount(this.props.getTokenInfo);
+
+    const { unitOfAccountSetting } = this.props;
+    return unitOfAccountSetting.enabled
+      ? (
+        <>
+          <div className={styles.fees}>+
```

Comment:
Seems not necessary to put a plus sign before the fee?

Reply:
We have it in the `WalletSendConfirmationDialgo.js` and `HWSendConfirmationDialog.js`

<img width="419" alt="Screen Shot 2022-01-30 at 3 58 01 AM" src="https://user-images.githubusercontent.com/72753578/151683776-966a6ad8-d624-4b11-b15e-c838b6cac275.png">